src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -2916,11 +2918,11 @@ public Mode3D get3DLayout() {
 			case "oulf":
 			case "top-bottom (left eye first)":
 				isAnaglyph = false;
-				return Mode3D.OUL;
+				return Mode3D.ABL;
 			case "ourf":
```

Comment:
@valib All the `case "ourf"` should be renamed, isn't it ?
Yes, i know it's just a little "refreshing" / "updating", but since you're on it i thought that it was worst mention it

**P.S.** Don't worry, i shouldn't review it more :wink: